### PR TITLE
Announce Cloud Haskell 0.2.2

### PR DESCRIPTION
**Pull request by qnikst**

### CODE REVIEW

File: _posts/2015-06-17-cloud-haskell-0.2.2.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+category: announcements
+tags: [announcement, distributed-process]
+title: "[ANNOUNCE]  Cloud Haskell 0.2.2"
+teaser: A bug-fix version of Cloud Haskell stack has been uploaded to hackage.
+author: Tim Watson
```
Comment (facundominguez) [inline _posts/2015-06-17-cloud-haskell-0.2.2.md]: Tim?

File: _posts/2015-06-17-cloud-haskell-0.2.2.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+link: https://groups.google.com/forum/#!topic/parallel-haskell/x3y6XNpTEFw
```
Comment (facundominguez) [inline _posts/2015-06-17-cloud-haskell-0.2.2.md]: The distributed-haskell list is a better source maybe.

**Comment by qnikst** [comment]
Fixed both.

**Comment by mboes** [comment]
The author on the post should be the name of the maintainer who did the release.

**Pull request merged**